src/main/scala/gameengine/GameEngine.scala

```diff
@@ -0,0 +1,109 @@
+package gameengine
+
+import characters.{GameBoard, Person, Player, Resources}
+
+import scala.Console.println
+import scala.collection.mutable.ListBuffer
+import scala.io.StdIn.readLine
+import scala.util.Random
+
+class GameEngine {
+  val resources = new Resources
+  private var questionList: List[String] = resources.listOfQuestions
+  private val random = new Random()
+  // Instantiate the game board
+  val gameBoard = new GameBoard(resources.charactersList)
+  // create firstPlayer
+  private val firstPlayer: Player = createPlayerAndAssignGameBoard()
+  private var continuedPlaying: Boolean = true
+
+  /**
+   * Create an object of player by assigning name, gameBoard and secret characters
+   * @return player object
+   */
+  def createPlayerAndAssignGameBoard(): Player = {
+    val player = new Player(name = "Player1", gameBoard = gameBoard.gameBoardForPlayer,
+      selectRandomCharacter(resources.charactersList))
+    player
```

Comment:
When making a person the instantiation of the class is great but you do not need to assign it to a val and then return the val. Just make the person as you have with new Person(....
and that is it as that is the last thing you are doing in this method that is what this method will return (a Person).

Reply:
We have removed player as the return